java/code/src/com/suse/manager/maintenance/rescheduling/FailRescheduleStrategy.java

```diff
@@ -32,9 +32,15 @@
     private static final Logger LOG = Logger.getLogger(CancelRescheduleStrategy.class);
 
     @Override
-    public RescheduleResult reschedule(User user, Map<Action, List<Server>> actionServer,
+    public RescheduleResult reschedule(User user, Map<Action, List<Server>> actionsServers,
             MaintenanceSchedule schedule) throws RescheduleException {
 
+        if (actionsServers.isEmpty()) {
+            RescheduleResult result = new RescheduleResult(getType().getLabel(), schedule.getName(), actionsServers);
+            result.setSuccess(true);
+            return  result;
+        }
+
```

Comment:
Would it be possible to test this case in a unit test?

Reply:
You're right. I think it should be possible. I'll investigate and add one if so.